src/Veldrid/BackendInfoOpenGL.cs

```diff
@@ -18,6 +19,51 @@ internal BackendInfoOpenGL(OpenGLGraphicsDevice gd)
             _gd = gd;
         }
 
+        /// <summary>
+        /// Gets the Version string of this OpenGL implementation.
+        /// </summary>
+        /// <remarks>
+        /// The string begins with a version number. The version number uses one of these forms:
+        /// <list type="bullet">
+        ///   <item>
+        ///     <description>major_number.minor_number</description>
+        ///   </item>
+        ///   <item>
+        ///     <description>major_number.minor_number.release_number</description>
+        ///   </item>
+        /// </list>
+        /// <para>
+        /// Vendor-specific information may follow the version number.
+        /// Its format depends on the implementation, but a space always separates the version number and the vendor-specific information.
+        /// </para>
+        /// </remarks>
+        public string Version => _gd.Version;
+
+        /// <summary>
+        /// Gets the Shader Language Version string of this OpenGL implementation.
+        /// </summary>
+        /// <remarks>
+        /// The string begins with a version number. The version number uses one of these forms:
+        /// <list type="bullet">
+        ///   <item>
+        ///     <description>major_number.minor_number</description>
+        ///   </item>
+        ///   <item>
+        ///     <description>major_number.minor_number.release_number</description>
+        ///   </item>
+        /// </list>
+        /// <para>
+        /// Vendor-specific information may follow the version number.
+        /// Its format depends on the implementation, but a space always separates the version number and the vendor-specific information.
+        /// </para>
+        /// </remarks>
+        public string ShadingLanguageVersion => _gd.ShadingLanguageVersion;
+
+        /// <summary>
+        /// Gets a collection of available OpenGL extensions.
+        /// </summary>
+        public OpenGLExtensions Extensions => _gd.Extensions;
```

Comment:
See other comment, but I think this should just be a `List<string>`, or a `ReadOnlyCollection<string>`, however similar things are being exposed. I don't think `OpenGLExtensions` itself needs to become public.